### ISSUE TITLE
Create default services

### ISSUE THREAD

**Issue by ccdarvin** [ccdarvin/vet.cloudbit.app#14]
Create a json with ICD (International Classification of Diseases) for vet

**Comment by ccdarvin** [comment]
For default data we must create a field in tenant, this field be un json with two values 
example
```
default: {
  <name>: {
    is_loaded: <bool>,
   can_reprocessed: <bool>,
  date: timezone
  }
}
```

**Comment by ccdarvin** [comment]
closed for benefit of #33
